pupil/views/partials/_question_content.ejs

```diff
@@ -1,3 +1,3 @@
-<main id="content" class="js-content" role="main">
+<main id="content" class="js-content">
```

Comment:
This roles were flagged as invalid markup. I will review roles in general as part of accessibility.